packages/db/index.js

```diff
@@ -836,40 +862,45 @@ export class DBClient {
 
     let query = this._client
       .from(psaPinRequestTableName)
-      .select(pinRequestSelect)
+      .select(listPinsQuery)
       .eq('auth_key_id', authKey)
       .order('inserted_at', { ascending: false })
 
-    if (opts.status) {
-      query = query.in('content.pins.status', opts.status)
-    }
-
-    if (opts.cid) {
-      query = query.in('source_cid', opts.cid)
-    }
-
-    if (opts.name && match === 'exact') {
-      query = query.like('name', `${opts.name}`)
-    }
-
-    if (opts.name && match === 'iexact') {
-      query = query.ilike('name', `${opts.name}`)
-    }
+    if (!Object.keys(opts).length) {
+      query = query.eq('content.pins.status', 'Pinned')
```

Comment:
Should `Pinned` be some kind of enum?

Reply:
That's a really good point, I thought about it as well.
But given we already have statuses enums (that don't quite do for our case though) I'm afraid adding another one would be a bit confusing. 
So If it's ok with you I'd leave it out this specific ticket.